### ISSUE TITLE
Duplicate Display DNN Copyright Setting

### ISSUE THREAD

**Issue by mikesmeltzer** [dnnsoftware/Dnn.AdminExperience#940]
## Description of bug

The Persona Bar gives 2 options, in different places, to control whether or not you want to inject the standard DNN Copyright (HTML comment) into every page on the site.

This is redundant to have 2 options to toggle the this behaviour and should be limited to one place. Additionally, the labeling of the 2 options is different which will cause even more confusion.

## Steps to reproduce

The options can be found at (screenshots below):

1.  Settings > Site Settings > Site Behavior > Privacy > Display Copyright

2.  Settings > Security > More > More Security Settings > Show Copyright Credits

## Expected result

The functionality should be revised to be accessible from only one place. The placement within the PersonaBar should be revisited to ensure that it is in the most appropriate spot.

## Screenshots

![image](https://user-images.githubusercontent.com/39993943/57696274-c48e3680-7626-11e9-9a06-cffada53b058.png)

![image](https://user-images.githubusercontent.com/39993943/57696981-8d208980-7628-11e9-9904-8d95c90c4ebc.png)

## Affected version

* [x] 9.3.2
* [x] 9.3.1

Other versions of DNN 9.* may also be affected.

**Comment by sleupold** [comment]
good catch, IMO it is security issue (hackers get to know, which CMS is used) and should be removed from privacy. It should be set to "off" by default.

**Comment by bdukes** [comment]
If it's a security issue that should be off by default, should it just be removed altogther?

**Comment by mikesmeltzer** [comment]
I don't believe this to be a security issue per say because the entire markup of any DNN site is filled with DNN identifiers. Every content pane, module, etc. is prepended with DNN identifiers. This would, absolutely, give a bit more obscurity to what you are running but not very much due to how the pages are built today.

Outside of identification, this adds unnecessary bloat to every page on the site. It is part of my standard checklist to turn this off for every site I work on to reduce page bloat.

Is there any benefit in having it? I'm unsure if search engines crawl comments, if they do it may help give some more exposure to the DNN / DotNetNuke terms but other than that I see little benefit in having it.  

I am for having it removed all together from both the Persona Bar and the underlying Platform functionality. Thoughts?

**Comment by sleupold** [comment]
the original intention was for visibility of the DNN project and it is nice for demo sites, but besides, we always turn it off for our clients. IMO it would be sufficient to have a copyright and link in the default website template.

**Comment by mikesmeltzer** [comment]
@sleupold I agree and think it should just be removed. 

I will create an RFC on the platform repo to have the copyright injection removed and see if there are anymore thoughts on it that we might be missing.

**Comment by ohine** [comment]
@mikesmeltzer, we can easily switch this issue over to an RFC if that's easier. I've got no issues with removing that setting completely from the platform.

**Comment by mikesmeltzer** [comment]
@ohine I've already created an RFC on the platform repository for this but thanks :)

**Comment by valadas** [comment]
I am also for completely removing this, both the feature and the UI for it, just not show the copyright at all.

**Comment by valadas** [comment]
The related issue in Platform is https://github.com/dnnsoftware/Dnn.Platform/issues/2783

**Comment by mitchelsellers** [comment]
Lets make this easy....remove it!

I've wanted it gone for years....

**Comment by valadas** [comment]
I am working on this now.
